src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -172,12 +172,17 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
       });
       const { databaseAccount: account, subscriptionId, resourceGroup } = userContext;
       if (!userContext.features.enableAadDataPlane) {
```

Comment:
This looks like it's making the list keys call, even if we already have them ? i.e. should we be skipping this if userContext.masterKey is already set ?